swmr_tools/Dask_Example.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
```

Comment:
Is this used and in the correct place?

Reply:
Dask_Example.py has been removed, it was not meant to be in the final repo. It was just where I was playing around with dask.